docs/ref/forms/fields.txt

```diff
@@ -380,6 +380,12 @@ For each field, we describe the default widget used if you don't specify
        If ``True`` (default), the value will be stripped of leading and
        trailing whitespace.
 
+    .. attribute:: empty_value
+
+       .. versionadded:: 1.11
+
+       The value to use to represent "empty". Defaults to the empty string.
```

Comment:
Maybe you could add another sentence giving an example use case for customizing it.